docs/quickstart.md

```diff
@@ -0,0 +1,249 @@
+
+    A general recommendation for these values is provided below:
+
+     - **Default idle timeout**: `30 Minutes`
+     - **Default retention period**: `1 Day`
+
+Codespaces can be deleted at anytime from the [Codespaces section of GitHub](https://github.com/codespaces).
+
+### Machine Types
+
+Different machine types are available within Codespaces. As a general rule, the greater the number of CPU cores, the greater number of hours that will be consumed during a Codespace's runtime.
+
+[GitHub's Pricing for Paid Usage](https://docs.github.com/en/billing/managing-billing-for-your-products/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#pricing-for-paid-usage) chart provides the details of the `usage multiplier` for each available machine type.
+
+For example, as of the writing of this guide:
+
+- :fontawesome-solid-microchip: `2 Core` Machine Types will consume `2` hours for every hour of runtime
+- :fontawesome-solid-microchip: `8 Core` Machine Types will consume `8` hours for every hour of runtime
+- :fontawesome-solid-microchip: `16 Core` Machine Types will consume `16` hours for every hour of runtime
+
+Some Arista Community Labs make use of larger machine types, such as the :fontawesome-solid-microchip: `16 Core` option.
+
+??? question "My GitHub Account doesn't have access to a larger :fontawesome-solid-microchip: `8 Core` or :fontawesome-solid-microchip: `16 Core` machine types?"
+    By default, some larger machine types may not be available for use. In order to resolve this, a ticket can be opened with [GitHub support](https://support.github.com/contact?source=subtitle&tags=rr-general-technical) requesting access to these larger machine types.
+
+    Listed below is a template that can be used for this request:
+
+    ```yaml
+    Hello - Can the 8-core and 16-core codespace machine types please be enabled for my account?
+    The default 2-core and 4-core machines lack the necessary resources for my use cases.
+    If additional information is needed, please let me know. Thanks!
+    ```
+
+    Once completed, select **Create a Ticket**
+
+## Starting the Lab
+
+!!! note
+    Throughout this guide, the [EVPN Domain A](https://codespaces.new/aristanetworks/aclabs/tree/main?quickstart=1&devcontainer_path=.devcontainer%2Ftechlib-vxlan-domain-a%2Fdevcontainer.json) lab associated with the [EVPN/VXLAN Deployment Guide](https://tech-library.arista.com/data_center/evpnvxlan/deployment_guide/) on [Arista's Tech Library](https://tech-library.arista.com) will be used as a reference point. All community labs hosted via Codespaces will follow the same process.
+
+Once a lab has been launched via it's respective 'Start Lab' button, a `Create Codespace` window will be opened via a web browser:
+
+<figure markdown>
+![Create Codespace](assets/img/aclabs-quickstart-1.png "Create Codespace"){ width=500px }
```

Comment:
All screenshots must be light version or on a gray background. Dark black is hard to read in the light MkDocs mode and also very low contrast with the current Mkdocs dark mode background. We can fix that later, after release. But at least we have to add `{ .opacity09 }`. Example: `![arista profile](assets/img/arista-profile.png){ .opacity09 }` - this slightly improves the match between background and the image without breaking image quality.

Reply:
For this specific case there is a light version already: https://aclabs.arista.com/assets/img/lab-enter-token.png

Reply:
Also all image names must be specific to identify them easily in the future. We can do that as step 2 however, as I'll be updating the structure anyway once we start templating.

Reply:
Image updated to `lab-enter-token.png`